build.gradle

```diff
@@ -1,9 +1,23 @@
 /**
  * This is the main gradle build file for the Azure DevOps IntelliJ Plugin.
  */
+buildscript {
+    ext {
+        rdGenVersion = '0.193.37'
```

Comment:
Yes, it is compatible. It is just a version of Rd generator, we doesn't bundle it and doesn't have any runtime dependency on it. I was testing the plugin in IDEA 2018.1 (the oldest we support) and Rider 2019.3 (in case of strange cross-rd runtime influences), and found none of the problems.